src/app/utils/ReactForm.js

```diff
@@ -121,7 +124,13 @@ export default function reactForm({
             }
 
             instance.setState({ [fieldName]: v }, () => {
-                setFormState(name, instance, fields, validation);
+                setFormStateDebounce(
```

Comment:
why should we debounce the form state setter, rather than [the validation function in Transfers.jsx](https://github.com/steemit/condenser/blob/master/src/app/components/modules/Transfer.jsx#L172)?

Reply:
(asking because this change will affect all users of our ReactForm module)

Reply:
I considered that.
The validation routine expects validators to be synchronous calls (and if well written, side-effect free).
Adding a debounce at this level keeps us from having to re-engineer ReactForm to handle validation calls that may never complete while still handling sync calls; or handle async only and rewriting all the validators to support async behavior.

This lets us keep the same api, gives us improved performance across all forms (reduced number of validations in case of other expensive validations like memo), and keeps our code change minimal.

Reply:
Also, I'm not saying I like the way ReactForm does things. But this came out as the best solution to the situation.

Reply:
cool, this makes sense to me. debouncing the setState rather than the validation call feels dirty, but this will get us by for now.

Reply:
The other thing I would say here is that the debounce function could take whatever function we want as an argument...